CHANGELOG.md

```diff
@@ -302,6 +302,8 @@ New features:
 
   To include global styles, you can set `$govuk-global-styles` variable to `true`.
 
+- Add override classes to set `display` property to `block`, `inline` and `inline-block` (PR [#694](https://github.com/alphagov/govuk-frontend/pull/652))
```

Comment:
Looks like the new feature is being added to the previous release

Reply:
yeah fixed now